flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/Execution.java

```diff
@@ -107,7 +108,7 @@
 	private static final AtomicReferenceFieldUpdater<Execution, ExecutionState> STATE_UPDATER =
 			AtomicReferenceFieldUpdater.newUpdater(Execution.class, ExecutionState.class, "state");
 	
-	private static final Logger LOG = ExecutionGraph.LOG;
+	private static final Logger LOG = LoggerFactory.getLogger(Execution.class);
```

Comment:
I have to admit that it confused me when parsing the log, because state changes of the `Execution` were reported from the `ExecutionGraph` logger. When I tried to pinpoint the exact location of the logging statement, I searched for the logging string in the `ExecutionGraph` but couldn't find it. But I also see your point. If you want to, I can revert the change.